dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -448,12 +447,50 @@ HRESULT WINAPI CDesktopFolder::CompareIDs(LPARAM lParam, PCUIDLIST_RELATIVE pidl
 {
     bool bIsDesktopFolder1, bIsDesktopFolder2;
 
+    if (LOWORD(lParam) >= DESKTOPSHELLVIEWCOLUMNS)
+    {
+        ERR("Got invalid column pointer %Ix\n", LOWORD(lParam));
```

Comment:
Why `%Ix`?

Reply:
I followed code in CDesktopFolder.cpp for the reasons given above.
https://github.com/reactos/reactos/blob/dd54cd32cf02254e9fc9ef78eb544bd580cd8787/dll/win32/shell32/folders/CDesktopFolder.cpp#L453

Reply:
> I followed code

No: `LOWORD()` returns a `WORD`, not a `*_PTR`.

Reply:
This code has been removed and its functionality moved into CFSFolder.cpp as noted above. Thanks.